ports/mbedTLS/azure_iot_jws_mbedtls.c

```diff
@@ -224,7 +225,7 @@ static AzureIoTResult_t prvJWS_RS256Verify( uint8_t * pucInput,
 
     if( lMbedTLSResult != 0 )
     {
-        AZLogError( ( "[JWS] mbedtls_rsa_import_raw res: %i", lMbedTLSResult ) );
+        AZLogError( ( "[JWS] mbedtls_rsa_import_raw res: %i", ( int ) lMbedTLSResult ) );
```

Comment:
should all the casts in this PR be int_32's/uint_32's to be more explicit?

Reply:
Since that's logging, I'm not so particular about standardizing those. But I will if we have a majority of votes :)

Reply:
Ok, I'll make the changes.

Reply:
Done.